tests/Type/Nette/data/componentModel.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+class SomeControl extends Nette\Application\UI\Control {
```

Comment:
1) This file needs a unique namespace
2) Please add a use statement for `function PHPStan\Testing\assertType`.

Reply:
@ondrejmirtes Done